layouts/container_footer.html

```diff
@@ -2,6 +2,6 @@
 
 <footer>
   <p class="pull-left">
-    &copy; Prometheus Authors 2016
+    &copy; Prometheus Authors 2017
```

Comment:
Alright, let's solve this issue once and for all with this PR: `<%= Date.today.year %>`.